portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,104 @@
+    autoStart: true,
+    cursor: '',
+    strings: []
+  });
 }
+
+// TEXT
+/**Heading text */
+const heading_text = "<div class = 'row'> <h1> Obi Abii's Portfolio </h1> </div>"; 
+
+/**About me blurb */
+const about_me_text = "<div class = 'row'> <p>Hi, My name is Obi and this is my <span class='emphasis'>portfolio! <br></span></p>\
+<p>I'm a CS Major at Cornell, currently interning at Google for the Summer of 2020. My main career focus is <span class='emphasis'>software engineering</span>. \
+I am particularly interested in <span class='emphasis'>Machine learning.</span></p></div>";
+
+/**blurb containg information about the languages I know */
+const language_blurb = "<div class = 'row'> <h2>languages</h2></div><div class = 'row'> <p> I have experience with the following languages</p></div>\
+  <!-- the list of languages with icons -->\
+  <div class='row row-cols-4 justify-content-center text-center emphasis align-middle'>\
+    <div class='col col-2'>\
+      <div class='col'><span class='iconify' data-icon='simple-icons:ocaml' data-inline='false'></span></div>\
+      <div class='col col-lg-2'> Ocaml</div>\
+    </div>\
+    <div class='col col-2 '>\
+      <div class='col'><i class='fab fa-html5'></i></div>\
+      <div class='col'> HTML</div>\
+    </div>\
+    <div class='col col-2 '>\
+      <div class='col'><i class='fab fa-css3'></i></div>\
+      <div class='col'>CSS</div>\
+    </div>\
+    <div class='col col-3 '>\
+      <div class='col'><i class='fab fa-js-square'></i></div> \
+      <div class='col'>JavaScript</div>\
+    </div>\
+    <div class='col col-2 '>\
+      <div class='col'><i class='fab fa-node-js'></i></div>\
+      <div class='col'> NodeJS</div>\
+    </div>\
+    <div class='col col-2 '> \
+      <div class='col'><i class='fab fa-java'></i></div>\
+      <div class='col'> Java</div>\
+    </div>\
+    <div class='col col-2 '> \
+      <div class='col'><i class='fab fa-python'></i></div>\
+      <div class='col'> Python</div>\
+    </div>\
+  </div>"
+
+  // <div class='col col-2'> \
+  //   C/C++ \
+  // </div>\
+
+const projects_blurb = "<div class = 'row'> <h2>projects</h2></div><div class = 'row'> <p> Below is a selection of some of my favorite projects.</p></div>"
+
+
+// DIV_ID
+/** The id of the div where the text will be placed */
+const console_id = 'console';
+
+/** List of project_ids */
+var projects = ['project_0', 'project_1', 'project_2']
+
+// TYPEWRITERS
+/**The names of the various typewriters I will use */
+var console = typify(console_id);
+
+// FUNCTIONS
+/**Make projects visible */
+function addProject(div_id){
+  document.getElementById(div_id).style.visibility = "visible";
+}
+
+
+
+/**print the initial text blurbs */
+
+console
+  .typeString(heading_text) //print heading
+  .pauseFor(300)
+  .typeString(about_me_text) //print about_me section
+  .pauseFor(300)
+  .typeString(language_blurb) //print language_section
+  .pauseFor(300)
+  .typeString(projects_blurb) //print project header
```

Comment:
I would lean towards removing these "print ... " comments. They don't really say anything the code doesn't already.